SP-3259 (#19)
lsst/survey-strategy_lsst_io: progress/index.rst

.. This section should provide a brief, top-level description of the page.

:doc:`LSST <lsst/index>`
#########################

Current status: Rubin Observatory has begun the 10-year Legacy Survey of Space and Time (LSST)
as of June 29, 2026. The :doc:`baseline survey strategy <../baseline/index>`
is in place.

`Nightly Scheduler Reports <https://s3df.slac.stanford.edu/data/rubin/sim-data/schedview/reports/>`_
(updated daily) provide a summary of observations taken,
along with pre-night simulations of upcoming nights and a
comparison of the actual visits against predictions.

.. toctree::
   :maxdepth: 2
   :titlesonly:
   :glob:

   lsst/index

:doc:`Pre-LSST <prelsst/index>`
################################

Following the handover from the Construction Project in late October 2025,
Rubin Observatory was in a period of early operations optimization known
colloquially as "Pre-LSST".

> Observations during this period, October 2025 to June 2026, have been a mixture of

Reviewer: ```suggestion
Observations during this period, October 2025 to June 2026, were a mixture of
```
